### PR TITLE
Updated en.json

### PR DESCRIPTION
**Pull request by Zezoo123**
Solves issue #39

### CODE REVIEW

File: custom_components/solarfocus/translations/en.json [hunk 1]
```diff
@@ -61,12 +61,12 @@
...
     "sensor": {
       "fm_state" : {
-        "state" : {
-          "0": "Vorlauffühler nicht angeschlossen",
-          "1": "Pumpe ausgeschaltet",
-          "2": "Pumpe eingeschaltet",
-          "3": "Handbetrieb aktiv",
-          "4": "Handbetrieb deaktiviert"
+        "state": {
+          "0": "preliminary sensor not connected",
```
Comment (LavermanJJ) [inline custom_components/solarfocus/translations/en.json:65]: Should we translate "Vorlauffühler" rather with "Supply sensor"? Similar to "Vorlauftemperatur" -> "Supply temperature"? WDYT?

**Review by LavermanJJ — CHANGES_REQUESTED** [review]
Thanks for providing the PR! 👍 
Would you mind, capitalizing the first word:
"manual mode activated" --> "**M**anual mode activated"

File: custom_components/solarfocus/translations/en.json (hunk 1 above)
Comment (lein1013) [reply custom_components/solarfocus/translations/en.json:65]: I vote for  "lead water temperature sensor" not connected or "lead temperature sensor"

**Comment by LavermanJJ** [comment]
I'll merge it like it is, and fix it on master

**Pull request merged**